gridstatus/tests/base_test_iso.py

```diff
@@ -230,7 +231,11 @@ def test_get_load_today(self):
 
     def test_get_load_forecast_historical(self):
         test_date = (pd.Timestamp.now() - pd.Timedelta(days=14)).date()
-        forecast = self.iso.get_load_forecast(date=test_date)
+        test_fcast_type = ERCOTSevenDayLoadForecastReport.BY_FORECAST_ZONE
```

Comment:
this is the base class that is used across all the tests for each individual iso. you want to update `test_ercot.py`. Because the return value for ercot is no different than the isos, you'll have to override this test on `TestErcot` by defining it